README.md

```diff
@@ -1,3 +1,116 @@
-# whisper
+![alt text][logo]
+# Whisper subprotocol [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
 This repository gathers various whisper-related tools
+
+## Contents
+
+- [Whisper SubProtocol](#whisper-subprotocol)
+  - [Implementations](#implementations)
+  - [Specifications / White Papers](#specifications--white-papers)
+- [Applications](#applications)
+  - [Desktop Interfaces](#desktop-interfaces)
+  - [Web Interfaces](#web-interfaces)
+  - [Mobile applications](#mobile-applications)
+  - [Explorers](#explorers)
+  - [Misc](#misc)
+- [Developer Resources](#developer-resources)
+  - [Libraries](#libraries)
+  - [Tutorials](#tutorials)
+  - [Example Projects](#example-projects)
+- [Learning Resources](#learning-resources)
+  - [Talks](#talks)
+  - [Books](#books)
+- [Community](#community)
+  - [Telegram](#telegram)
+- [Contribute](#contribute)
+- [License](#license)
+
+
+## Whisper SubProtocol
+
+### Implementations
+
+Implementations of the Whisper subprotocol
+
+### Specifications / White Papers
+
+- [EIP-627: Whisper Specification][https://github.com/ethereum/EIPs/blob/master/EIPS/eip-627.md]
+
+- [Formal spec][https://github.com/ethereum/whisper/spec/WHISPER.md]
+
+### Wiki guides
+
+- [Whisper wiki][https://github.com/ethereum/go-ethereum/wiki/Whisper]
+
+## Applications
+
+### Desktop Interfaces
+
+- [wnode][https://github.com/ethereum/go-ethereum/wiki/Diagnostic-Tool-wnode]
+
+### Web Interfaces
+
+### Server/Router Clients
+
+### IoT Clients
```

Comment:
Do you have examples in mind? Otherwise, I think that we should remove this section not to give an incorrect impression about the current scalability of Whisper.